build

```diff
@@ -149,6 +154,7 @@ if [ -d ./cache ]; then
   rsync -a /scratch/state/ ./cache/state
 fi
 
+mkdir -p image
```

Comment:
This could alternatively be wrapped in an `if [ -d ./image ]` if we want to allow both to be optional per my previous comment.

Reply:
The tricky thing here is that `rootfs` actually relies on the output of `img save`, so if _either_ of them are specified, we have to output an `image.tar`. The `mkdir -p` solves for the case where `rootfs` is specified but `image` is not.

We can still get the behavior of both being optional by exiting between the build and the save in the case that neither is defined. I'll have that change pushed up in a minute

Reply:
Ah good point!